src/main/java/com/browserstack/automate/ci/common/BrowserStackBuildWrapperOperations.java

```diff
@@ -129,6 +130,15 @@ public void buildEnvVars(Map<String, String> env) {
             }
         }
 
+        String grr_region = env.get(BrowserStackEnvVars.GRR_JENKINS_KEY);
+        if ( grr_region != null && Constants.GRR_AUTO_REGION_VS_APIURL.containsKey(grr_region.toLowerCase()) ) {
```

Comment:
can we add else block to log what are the valid values in case user configures incorrect value in environment variable.

Reply:
Handled it.